tracing/src/lib.rs

```diff
@@ -1001,6 +1001,7 @@ pub mod __macro_support {
     use crate::{collect::Interest, Metadata};
     use core::fmt;
     use core::sync::atomic::{AtomicU8, Ordering};
+    pub use core::{concat, format_args, iter::Iterator, option::Option};
```

Comment:
Will it be simpler to just `pub use core;` so no need to collect and maintain every references in macros?

Reply:
We could do that... the only thing is that the full paths will be quite long in the `macros.rs` file (e.g.: `$crate::__macro_support::iterator::Iterator::next()`).